### DISCUSSION TITLE
Oqtane’s Modules & DotNet Version Upgrades

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#3331] [General] [answered]
How do I upgrade all of my Oqtane’s modules and dotnet versions in one click?

**Comment by sbwalker** [comment, marked as answer]
there is no automated method to upgrade your dotnet versions, as the references are embedded in every *.cspro file. If you want to use a newer version of .NET you will need to manually update the references.
